### PR TITLE
Birefringence2

### PR DESCRIPTION
**Pull request by cg-laser**
this is a copy of #319 to fix a broken merge commit. Please refer to #319 for all previous discussion

### CODE REVIEW

File: NuRadioMC/test/SignalProp/T08test_emitter_birefringence.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    event_traces = np.vstack(list_trace)
+    list_traces.append(event_traces)
+
+traces = np.dstack(list_traces)
+traces = np.transpose(traces, (2, 0, 1))
+
+reference_array = np.load('reference_emitter.npy')
+
+testing.assert_allclose(traces, reference_array, atol=2e-4  * units.V / units.m, rtol=1e-7)
```
Comment (cg-laser) [inline NuRadioMC/test/SignalProp/T08test_emitter_birefringence.py]: isn't this a fairly large absolute tolerance as the efield strengths is typically micro volts?

Comment (nilsphysics) [reply NuRadioMC/test/SignalProp/T08test_emitter_birefringence.py]: I agree. When I'm running this test locally there is no issue with setting this tolerance to much stricter levels. However, when pushing it to GitHub there are deviations between the results. I tried to look into what dependency versions are installed on the remote server but I couldn't really find a difference. Let me know if you have an idea.

**Comment by nilsphysics** [comment]
I think I now implemented all the final comments and requests.

**Review by nilsphysics — DISMISSED** [review]
From my side it would be ready to merge.

**Review by nilsphysics — APPROVED** [review]
From my side it would be ready to merge.

**Pull request merged**